readthedocs/config/config.py

```diff
@@ -399,26 +396,22 @@ def validate_python(self):
                         'python.extra_requirements',
                         self.PYTHON_EXTRA_REQUIREMENTS_INVALID_MESSAGE,
                         code=PYTHON_INVALID)
-                for extra_name in raw_extra_requirements:
-                    with self.catch_validation_error(
-                            'python.extra_requirements'):
-                        python['extra_requirements'].append(
-                            validate_string(extra_name)
-                        )
+                if not python['install_with_pip']:
+                    python['extra_requirements'] = []
```

Comment:
I don't think it should affect, but this checking will be new for V1.

Reply:
Yeah, we were doing this validation in the rtd code before.